### PR TITLE
Document property info exclude static properties

### PR DESCRIPTION
**Pull request by fbourigault**
Documentation for symfony/symfony#28775.

### CODE REVIEW

File: components/property_info.rst [hunk 1]
```diff
@@ -409,6 +409,22 @@ constructor. It supports return and scalar types for PHP 7.
     // Initializable information
     $reflectionExtractor->isInitializable($class, $property);
 
+Excluding static properties
```
Comment (nicolas-grekas) [inline components/property_info.rst]: maybe move this as a tip (and remove this title)?

File: components/property_info.rst [hunk 2]
```diff
@@ -409,6 +409,22 @@ constructor. It supports return and scalar types for PHP 7.
...
+...........................
+By default, the static properties are extracted. To get them excluded,
```
Comment (nicolas-grekas) [inline components/property_info.rst]: By default, the static properties are also extracted. To exclude them,[...]

File: components/property_info.rst [hunk 3]
```diff
@@ -409,6 +409,22 @@ constructor. It supports return and scalar types for PHP 7.
...
+you can use the ``exclude_static_properties`` context option.
+
+
+.. code-block:: php
+
+    use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+
+    $reflectionExtractor = new ReflectionExtractor();
+
+    $reflectionExtractor->getProperties($class, [
```
Comment (nicolas-grekas) [inline components/property_info.rst]: code example could maybe be reduced to this single call, given the context above?

**Comment by fbourigault** [comment]
Closing as code PR is now closed.

**Pull request closed without merging**